src/main/java/de/bonndan/nivio/model/Item.java

```diff
@@ -321,5 +321,24 @@ public String getAssessmentIdentifier() {
         return changes;
     }
 
+    public String[] getNetworks() {
```

Comment:
We don't need this getter/setter, networks come via labels and ItemApiModel can use them. Labeled interface also has some convenience methods to pick prefixed labels.